### PR TITLE
EZP-31784 - Autosave description

### PR DESCRIPTION
**Pull request by dabrt**
| Question      | Answer |
| ------------- | --- |
| JIRA Ticket   | EZP-31784 |
| Versions      | master |

A description of the Autosave feature

### CODE REVIEW

File: docs/publishing/publishing.md [hunk 1]
```diff
@@ -27,6 +27,17 @@ You will also have the option to create a new draft based on the currently publi
 
 Note that you cannot edit an unpublished draft created by another user.
 
+### Autosave
+
+eZ Platform comes with an Autosave feature that helps you preserve your work in an event of a failure while you edit a content item or product description.
```
Comment (DominikaK) [inline docs/publishing/publishing.md:32]: What do you think about:
```suggestion
You work is saved automatically. Autosave helps you preserve your progress in an event of a failure while you edit a Content item or product.
```
This way avoids mentioning a feature, in the spirit of "don't describe features, describe what the user can do".
Although the passive could be fixed further...

File: docs/publishing/publishing.md [hunk 2]
```diff
@@ -27,6 +27,17 @@ You will also have the option to create a new draft based on the currently publi
...
+To recover your work, open the most recent draft in the **Versions** tab of the content item or
+product description.
```
Comment (DominikaK) [inline docs/publishing/publishing.md:34]:
```suggestion
To recover your work, open the most recent draft in the **Versions** tab of the Content item.
```
If product is stored in the content tree, it is a Content item, no need to mention it explicitly.

File: docs/publishing/publishing.md [hunk 3]
```diff
@@ -27,6 +27,17 @@ You will also have the option to create a new draft based on the currently publi
...
+Alternatively, open the most recent draft of your work on the **My dashboard** page, the **Drafts** table.
+
+The feature is enabled by default, and set to save a draft of your work every 60 seconds.
```
Comment (DominikaK) [inline docs/publishing/publishing.md:37]:
```suggestion
Autosave is enabled by default, and set to save a draft of your work every 60 seconds.
```

File: docs/publishing/publishing.md [hunk 4]
```diff
@@ -27,6 +27,17 @@ You will also have the option to create a new draft based on the currently publi
...
+You can toggle Autosave or change the time between saving attempts in **User settings**, by changing
```
Comment (DominikaK) [inline docs/publishing/publishing.md:38]:
```suggestion
You can toggle autosave or change the time between saving attempts in **User settings**, by changing
```

File: docs/site_organization/working_with_page.md [hunk 5]
```diff
@@ -24,6 +24,11 @@
     Before publishing or saving the Page, edit its title and description.
     Switch to the Field view using the [Page Builder toolbar](#page-builder-toolbar).
 
+    !!! note "Autosave"
+
+        To help you preserve your work eZ Platform comes with an Autosave feature.
```
Comment (DominikaK) [inline docs/site_organization/working_with_page.md:29]:
```suggestion
        To help you preserve your work eZ Platform autosaves your progress.
```

File: docs/creating_content_basic.md [hunk 6]
```diff
@@ -31,6 +31,11 @@ Some Fields, such as *Relation* Fields (which link two Content items), or *Image
 require you to select a different Content item to link to.
 A *Location* Field is a point on the map. You can type the place name, enter its coordinates, or select it on the map.
 
+!!! note "Autosave"
+
+    To help you preserve your work eZ Platform comes with an Autosave feature.
```
Comment (DominikaK) [inline docs/creating_content_basic.md:36]:
```suggestion
    To help you preserve your work eZ Platform autosaves your progress.
```

**Review by DominikaK — CHANGES_REQUESTED** [review]

**Review by DominikaK — APPROVED** [review]

**Review by Nattfarinn — APPROVED** [review]

**Pull request merged**